swift/container/backend.py

```diff
@@ -1196,7 +1213,7 @@ def pivot_nodes_to_items(self, nodes):
     def build_pivot_ranges(self):
         ranges = list()
 
-        for node in self.get_pivot_ranges():
+        for node in sorted(self.get_pivot_ranges()):
```

Comment:
This does not feel right, I think there could be a feature of sqlite I am missing that would make this less painful (but I don't think order by is right either). Do you know of a better way to keep the items in order in the database?